src/sphinxcontrib/bibtex/__init__.py

```diff
@@ -48,7 +53,21 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     app.add_post_transform(BibliographyTransform)
     app.add_domain(BibtexFootDomain)
     app.add_directive("footbibliography", FootBibliographyDirective)
-    app.add_role("footcite", FootCiteRole())
+    app.add_role("footcite", FootCiteRole())"config",
+        )
+
+    # Catch bug in newer docutils
+    if docutils.__version_info__.major == 0 and docutils.__version_info__.minor >= 18:
```

Comment:
Two comments here:
1. What's the reason for not restricting this to minor < 20? We know the fix is applied to the current git version of docutils, so the next version will very likely include the fix. In that way only docutils needs to be updated.
2. Tuple comparison might simplify the logic slightly.

Reply:
> What's the reason for not restricting this to minor < 20?

That's true, I hadn't fully appreciated that the fix was already in `main`. Let's fix this.

Reply:
(2) yes, I was a bit twitchy about unpacking a tuple like this, but I doubt it will change.

(1) it's an assumption that might be wrong, rather than the guarantee that we'll need to update this package. Both are fine, we can always re-release this package if there turns out to be an issue.